web/src/lib/components/user-settings-page/partner-settings.svelte

```diff
@@ -0,0 +1,107 @@
+<script lang="ts">
+	import { UserResponseDto, api } from '@api';
+	import CircleAvatar from '../shared-components/circle-avatar.svelte';
+	import Close from 'svelte-material-icons/Close.svelte';
+	import Button from '../elements/buttons/button.svelte';
+	import PartnerSelectionModal from './partner-selection-modal.svelte';
+	import { handleError } from '../../utils/handle-error';
+	import { browser } from '$app/environment';
+	import { onMount } from 'svelte';
+
+	let partners: UserResponseDto[] = [];
+	let showAddUserModal = false;
+
+	$: {
+		if (browser) {
+			if (showAddUserModal) {
+				document.body.style.overflow = 'hidden';
+			} else {
+				document.body.style.overflow = 'auto';
+			}
+		}
+	}
```

Comment:
Scroll will stay disabled if the user opens the modal and goes to another page without closing it. This can probably be added to the modal component using lifecycle functions instead

Reply:
I moved this code to `BaseModal`, but then after some testing and thinking I decided to revert this scrollbar style changes in this PR at all. I don't like how centered content of the page (user account settings, for example) immediately jumps to the right after I hide a scrollbar